matrix/system_of_linear_equation_in_2_variables.py

```diff
@@ -0,0 +1,102 @@
+"""
+Theory:-
+    https://www.mathsisfun.com/algebra/systems-linear-equations-matrices.html
+    Cramer's rule for 2x2 matrix:-
+    https://www.chilimath.com/lessons/advanced-algebra/cramers-rule-with-two-variables
+    a1x + b1y = = d1
+    a2x + b2y = = d2
+"""
+
+
+def calculate_x_and_y(eq1: list[int], eq2: list[int]) -> None:
+    """
+    Solves the system of linear equation in 2 variables.
+    :param: eq1: list of 3 numbers
+    :param: eq2: list of 3 numbers
+    :return: String of result
+    input format : [a1, b1, d1], [a2, b2, d2]
+    d_matrix = [[a1, b1], [a2, b2]]
+    d is determinant of matrix d_matrix
+    dx_matrix = [[d1, b1], [d2, b2]]
+    dx is determinant of matrix dx_matrix
+    dy_matrix = [[a1, d1], [a2, d2]]
+    dy is determinant of matrix dy_matrix
```

Comment:
Many of these comments can be eliminated by using better variable names.
* d --> determinant or determinant_matrix
* dx --> determinant_x or determinant_matrix_x
* dx --> determinant_y or determinant_matrix_y